types/ldapjs/index.d.ts

```diff
@@ -512,7 +512,11 @@ export interface AttributeJson {
 }
 
 export class Attribute {
-    private type: string;
+    constructor(options?: {
+        type?: string,
+        vals?: any;
```

Comment:
Do you think it's worth adding more explicit types here?
`(string | Buffer)[]`

in reality works for other types too, and they're stringified.

Reply:
I was thinking it would be best to avoid the type definitions preventing the user from making use of the full featureset of the package, i.e. keeping the `any`.